compiler/rustc_error_messages/locales/en-US/rustc_mir_dataflow.ftl

```diff
@@ -0,0 +1,29 @@
+rustc_mir_dataflow_path_must_end_in_filename =
+    path must end in a filename
+
+rustc_mir_dataflow_unknown_formatter =
+    unknown formatter
+
+rustc_mir_dataflow_duplicate_values_for =
+    duplicate values for `{$name}`
+
+rustc_mir_dataflow_requires_an_argument =
+    `{$name}` requires an argument
+
+rustc_mir_dataflow_stop_after_dataflow_ended_compilation =
+    stop_after_dataflow ended compilation
+
+rustc_mir_dataflow_peek_must_be_place_or_ref_place =
+    rustc_peek: argument expression must be either `place` or `&place`
+
+rustc_mir_dataflow_peek_must_be_not_temporary =
+    dataflow::sanity_check cannot feed a non-temp to rustc_peek.
```

Comment:
Pre-existing but could you remove the below period? IIRC we generally don't add it in diagnostics.

```suggestion
    dataflow::sanity_check cannot feed a non-temp to rustc_peek
```